core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -592,8 +592,14 @@ private[spark] class BlockManager(
     val locations = Random.shuffle(master.getLocations(blockId))
     for (loc <- locations) {
       logDebug(s"Getting remote block $blockId from $loc")
-      val data = blockTransferService.fetchBlockSync(
-        loc.host, loc.port, loc.executorId, blockId.toString).nioByteBuffer()
+      val data = try {
```

Comment:
With my experience, if you assign to a variable with a set of statements, you have to encapsulate those statements with a `{}`.....